sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/CustomValidationPolicy.java

```diff
@@ -76,17 +72,9 @@ private static String ungzip(byte[] rawBytes) {
             while ((read = in.read(data, 0, data.length)) != -1) {
                 baos.write(data, 0, read);
             }
-            return new String(baos.toByteArray(), StandardCharsets.UTF_8);
+            return baos.toString();
```

Comment:
```suggestion
            return baos.toString(StandardCharsets.UTF_8.name());
```

Reply:
intellij warns me to replace it with StandardCharsets.UTF_8, and then it caused the compilation error. i can ignore the warnings 😆